docs/user-guide/services.md

```diff
@@ -388,15 +388,64 @@ as of Kubernetes 1.0.
 
 ### Type NodePort
 
-If you set the `type` field to `"NodePort"`, the Kubernetes master will
+If you set the `spec.type` field to `"NodePort"`, the Kubernetes master will
 allocate a port from a flag-configured range (default: 30000-32767), and each
 Node will proxy that port (the same port number on every Node) into your `Service`.
 That port will be reported in your `Service`'s `spec.ports[*].nodePort` field.
 
-If you want a specific port number, you can specify a value in the `nodePort`
+```json
```

Comment:
It's not clear to a reader if this is the input JSON or output.  Since spec.ports[*].nodePort is not populated I assume it is input.  It might be more useful to show a snippet of input and a snippet ouf output, maybe using kubectl -o template ?